baselines/Google Chat Minimum Viable Secure Configuration Baseline v0.1.md

```diff
@@ -300,3 +300,72 @@ Agencies SHOULD configure DLP rules to block or warn on sharing files with sensi
 9.  Select the appropriate action to warn or block sharing, based upon the agency's individual requirements.
 10. In the **Alerting** section, choose a severity level, and optionally, check **Send to alert center to trigger notifications**.
 11. Review the rule details, mark the rule as **Active**, and click **Create.**
+
+## 7. Content Reporting
+
+This section covers the content reporting functionality.
+
+### Policies
+
+#### GWS.CHAT.7.1v0.1
+Chat Content Reporting SHALL be Enabled.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
+- Last Modified: February 13, 2024
+
+- MITRE ATT&CK TTP Mapping
+  - None
+
+#### GWS.CHAT.7.2v0.1
+All Conversation Types SHALL be Selected.
+
+- Rationale
+  - This ensures that users are able to report content for different categories such as sensitive or confidential information. This provides additional monitoring capabilities.
```

Comment:
IMO the rationale does not adequately address the policy. Recommend something like this:

```suggestion
  - Risky messages could potentially come through any of the different Chat conversation types. By selecting all conversation types, agencies ensure that their users are able to report risky messages regardless of the conversation type.
```